structure_tests/Makefile

```diff
@@ -19,22 +19,41 @@ VERSION_BUILD ?= 0
 
 VERSION ?= v$(VERSION_MAJOR).$(VERSION_MINOR).$(VERSION_BUILD)
 
+GOOS ?= $(shell go env GOOS)
+GOARCH = amd64
+ORG := github.com/GoogleCloudPlatform
+PROJECT := runtimes-common/structure_tests
+REPOPATH ?= $(ORG)/$(PROJECT)
+RELEASE_BUCKET ?= $(PROJECT)
+
+SUPPORTED_PLATFORMS := linux-$(GOARCH) darwin-$(GOARCH) windows-$(GOARCH).exe
+BUILD_PACKAGE = $(REPOPATH)
+
 BUILD_DIR ?= ./out
 BUCKET ?= structure-test
 UPLOAD_LOCATION := gs://${BUCKET}
 
-all: clean build
+$(BUILD_DIR)/$(PROJECT): $(BUILD_DIR)/$(PROJECT)-$(GOOS)-$(GOARCH)
+	cp $(BUILD_DIR)/$(PROJECT)-$(GOOS)-$(GOARCH) $@
 
-.PHONY: build
-build: structure-test
+$(BUILD_DIR)/$(PROJECT)-%-$(GOARCH): $(GO_FILES) $(BUILD_DIR)
+	GOOS=$* GOARCH=$(GOARCH) CGO_ENABLED=0 go test -c . -o $@ $(BUILD_PACKAGE)
 
-.PHONY: release
-release: structure-test
-	gsutil cp $(BUILD_DIR)/structure-test $(UPLOAD_LOCATION)/$(VERSION)/structure-test-amd64
+%.sha256: %
+	shasum -a 256 $< &> $@
+
+%.exe: %
+	mv $< $@
```

Comment:
i think this is an error in container-diff too, but this should probably be a `cp` instead of a `mv`

Reply:
Yeah, should be cp. I think @r2d4 just fixed this somewhere, maybe in minikube?